apps/site/lib/site_web/controllers/trip_plan_controller.ex

```diff
@@ -113,6 +113,18 @@ defmodule SiteWeb.TripPlanController do
     end
   end
 
+  defp get_route(link) do
+    if is_list(Enum.at(link.text, 0)) do
```

Comment:
This will throw an error when `link.text` does not happen to be a list, which happens sometimes on non-MBTA routes. One example link I ran into when planning a trip from South Station to Boston Logan Airport:

```
%Site.TripPlan.RelatedLink{
  icon_name: :massport_shuttle,
  text: "Massport schedules",
  url: "https://massport.com/"
}
```

Can we add a condition to check for/handle this case where text is a single string?

Reply:
Also! I'd suggest using `List.to_string()` here to get the text, e.g.

```elixir
[["Route ", "SL1"], " schedules"] |> List.to_string()
# returns "Route SL1 schedules"
```

Reply:
@thecristen should I be using `List.to_string()` for all the cases where link.text is an array ie.
```
["Green Line E", " schedules"],
[["Route ", "SL1"], " schedules"],
["View fare information"],
```

Reply:
@i0sea Yeah I'd say so!